app/_src/gateway/production/monitoring/healthcheck-probes.md

```diff
@@ -10,6 +10,34 @@ The readiness check endpoint returns a `200 OK` response when {{site.base_gatewa
 {:.note}
 > **Note:**  The readiness endpoint does not return detailed information about the node status.
 
+## Types of Health Checks
+
+For each {{site.base_gateway}} node, there exist two distinct health checks (also known as "probes"):
+
+* Liveness: This `/status` endpoint responds with a `200 OK` status if Kong is running. The request will fail either with a `500 Internal Server Error` or no response if Kong is not running. You can send a GET request to check the liveness of your {{site.base_gateway}} instance:
+  ```sh
+  # Replace `localhost:8100` with the appropriate host and port for
+  # your Status API server
+  
+  curl -i http://localhost:8100/status
+  ```
+
+* Readiness: This `/status/ready` endpoint responds with a `200 OK` status if Kong has successfully loaded a valid configuration and is ready to proxy traffic. The request will fail either with a `500 Internal Server Error` or no response if Kong is not ready to proxy traffic yet. You can send a GET request to check the readiness of your {{site.base_gateway}} instance:
+  ```sh
+  # Replace `localhost:8100` with the appropriate host and port for
+  # your Status API server
+  
+  curl -i http://localhost:8100/status/ready
+  ```
+
+These two types of health checks for {{site.base_gateway}} are modeled on how [Kubernetes defines](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/) health check probes.
+
+It is strongly recommended that a component (i.e. load balancer) perform the readiness health check before sending traffic. This ensures that a {{site.base_gateway}} node has not only successfully started up but has also finished loading up the configuration and is ready to receive proxy traffic. 
```

Comment:
```suggestion
We strongly recommend that a component (that is, a load balancer) perform the readiness health check before sending traffic. This ensures that a {{site.base_gateway}} node has not only successfully started up but has also finished loading up the configuration and is ready to receive proxy traffic. 
```
* "We recommend" is the phrasing we use in the docs (and I just found out that it wasn't in our style guide, so I've gone and added it).
* Avoid using latin shortforms.